lib/dependencies/HarmonyImportSpecifierDependency.js

```diff
@@ -131,7 +131,6 @@ class HarmonyImportSpecifierDependency extends HarmonyImportDependency {
 
 HarmonyImportSpecifierDependency.Template = class HarmonyImportSpecifierDependencyTemplate extends HarmonyImportDependency.Template {
 	apply(dep, source, runtime) {
-		super.apply(dep, source, runtime);
```

Comment:
This call was no-op

Reply:
You should still keep it. Maybe we add code to `HarmonyImportDependencyTemplate.apply` in future.

Reply:
Sure, I'll revert this